### DISCUSSION TITLE
I used this on Sonoma 14.5, if i update my software will i have to do the bypass again?

### DISCUSSION THREAD

**Discussion by reabo** [assafdori/bypass-mdm#18] [Q&A]
I cant find a clear or honest answer anywhere. Id this bypass last year for Sonoma 14.5 and avoided updates incase id have to run the bypass again.

Any one in a similar position and updated via system settings? Did you mac run as normal or did you need to do thebypass again ti keep it MDM free?

**Comment by Herores** [comment]
Is there any news? Can we do the same process with Version 15.2  ?

**Comment by Herores** [comment]
@reabo  All good! Works perfectly! 

https://www.youtube.com/watch?v=_k4J1DkEov4
